### PR TITLE
doc: add openssl generated certificate

### PR DESCRIPTION
**Pull request by JeyJeyGao**
Signed-off-by: Junjie Gao <junjiegao@microsoft.com>

### CODE REVIEW

File: docs/ca-signed-workflow.md [hunk 1]
```diff
@@ -81,14 +99,15 @@
    docker tag hello-world:latest $server/hello-world:v1
    docker push $server/hello-world:v1
    ```
-9. Sign the image with an external certificate bundle (`$certBundlePath`) including the intermediate certificates and a root certificate in PEM format. You may fetch the certificate bundle from your CA official website.
+9. Sign the image with an external certificate bundle (`$certBundlePath`) including the intermediate certificates and a root certificate in PEM format. You may fetch the certificate bundle from your CA official site.
+   > **Note** If you generated the certificate with `openssl` the certificate bundle is the root certificate `ca.crt`.
```
Comment (FeynmanZhou) [inline docs/ca-signed-workflow.md:103]:
```suggestion
   > **Note** If you have generated the certificate with `openssl` according to the above steps, the certificate bundle is the root certificate `ca.crt`.
```

File: docs/ca-signed-workflow.md [hunk 2]
```diff
@@ -55,19 +55,37 @@
...
    ```
-5. Please take `${certName}.csr` file to a trusted CA to sign and issue your certificate, or you can use `openssl` tool to sign it locally for testing.
-6. After you get the leaf certificate, you can merge the leaf certificate (`$leafCert`) to your Azure Key Vault:
+5. Please take `${certName}.csr` file to a trusted CA to sign and issue your certificate, or you can use `openssl` tool to sign it locally for testing. Here is an example by using `openssl`:
+   Create a private key and certificate for a root CA with `openssl`:
    ```sh
-   az keyvault certificate pending merge --vault-name $keyVault --name $certName --file $leafCert
+   openssl req -x509 -sha256 -nodes -newkey rsa:2048 -keyout ca.key -out ca.crt -days 365 -subj "/CN=Test CA" -addext "keyUsage=critical,keyCertSign"
+   ```
+   Create a config file for `openssl` to sign the leaf certificate:
```
Comment (FeynmanZhou) [inline docs/ca-signed-workflow.md:67]:
```suggestion
   Create a configuration file for `openssl`. It will be used to sign the leaf certificate:
```

**Review by FeynmanZhou — COMMENTED** [review]
PTAL at my comments. Thanks

File: docs/ca-signed-workflow.md (hunk 1 above)
Comment (JeyJeyGao) [reply docs/ca-signed-workflow.md]: Updated.

File: docs/ca-signed-workflow.md (hunk 2 above)
Comment (JeyJeyGao) [reply docs/ca-signed-workflow.md]: Updated.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**